docs/mentored_sprint_agenda.md

```diff
@@ -2,35 +2,66 @@
-- **1:45- 2:00** Break time! Move around.
-- **2:00- 2:45** Work time part 2! Remember to get review.
-- **2:45- 3:00** Wrap up! Time for giving feedback and info on upcoming events.
-- **3:00- 4:00 Optional** Celebrate! Nominate `alt text hall of fame` by category and vote. 
-Top voted ones and their authors live in infamy on the repo. This time is also spillover 
-for people still wanting to connect with others.
-
-## Questions in search of answers
-- Do we focus only on alt text, or descriptive content for other media as 
-well? The JupyterLab docs have a lot of videos.
-- Do we need resources for git and github, or is that out of scope for our 
-attendees?
-- Project Jupyter docs have less content and are a more manageable goal for 
-the sprint. Do enough people use them for this to do a helpful task?
-- What is a reasonable amount of time for a sprint? How long do we want this 
-to be?
+## October 9
+
+A hands-on event in Jupyter code bases that improve and document accessibility conditions.
+
+### Takeaways
+
+Attendees will become familiar with the different skills needed to impact the Jupyter ecosystem, especially around documentation and alt text.
+
+### Pre-event
+In order to participate in the sprint, attendees will need a GitHub account. No other knowledge of git or GitHub is required.
+
+### Schedule
+
+```mermaid
+gantt
+
+    dateFormat YYYY-MM-DD hh:mm
+    axisFormat  %H:%M
+
+    section icebreaker
+    meet and greet :meet, 2021-10-02 08:45, 15min
+    welcome, welcome, welcome :welcome, after meet, 15min
+        
+    section introductions
+    intro to jupyter :jupyter, after welcome, 10min 
+    wcag and accessibility tools :tools, after jupyter, 30min
+            
+    section breakouts
+    breakout topics :topics, after tools, 10min
+    breakout 1 :b1, after topics, 40min
+    breakout 2 :b2, after topics, 40min
+    summarize :regroup, after b1, 10min
+    past a11y issues/prs :past, after regroup, 10min
+    breakout 1 :b21, after past, 45min
+    breakout 2 :b22, after past, 45min
+        
+    section conclusion
+    final comments :end, after b21, 15min
+    
+    section hangout
+    hangout :hang, after end, 45min
+    
+```
+
+* Enter in breakout rooms (like you about to go on an amusement park ride)
+* Welcome, welcome, welcome
+* Intro to jupyter
+* Intro to accessibility tools (WAVE, JAWS, NVDA)
+    * Demo tools on documentation to show the problem (and solution?)
+* Breakout topics
+    * Writing alt text for [Project Jupyter documentation](https://jupyter.readthedocs.io/en/latest/) or other Jupyter ecosystem documentation (no git knowledge needed)
+    * Adding automated tests ([pa11y](https://pa11y.org/) and/or [Lighthouse](https://developers.google.com/web/tools/lighthouse/) to Project Jupyter documentation)
+* Breakouts
+    * Work on your chosen topic with smaller group
+* Break
+* A look at past accessibility PRs (community member)
+* Breakouts
+* Closing 
```

Comment:
Is this closing for regrouping and celebrate wins?

Reply:
Should probably include
- Current state of PR(s) open around this sprint. In an ideal world, we could click the merge button with everyone, but if it needs more review I think it will also be satisfying to just count the number or images described. If we have people working on testing, I think we will just have to see how far they get.
- Thanks for being here! Here's a feedback survey.
- You can keep up with future workshops on this repo, join other Jupyter accessibility efforts in jupyter/accessibility or meetings on the community calendar

Things I would love to include but have been removed for time in past drafts:
- Nominate alt text a team member wrote or that you reviewed to an ✨ alt text hall of fame ✨ for both kudos and to make a list of great alt text examples for future contributors.